phoenicis-blue.svg

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This should be in https://github.com/PhoenicisOrg/resources

Reply:
I will add it.

Reply:
Or move? We shouldn't have it in 3 different places.

Reply:
I think that it makes sense to have it somewhere in a submodule, so that it could be inserted anywhere (Readme, about window, etc..)

Reply:
What about phoenicis-javafx/src/main/resources/org/phoenicis/javafx/views/common/? At least we have the `phoenicis.png` there already so it would fit somehow.

Reply:
Good to me